src/component/Task/components/CompletedTask.js

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect } from 'react';
+import styled from "styled-components";
+import { getTasksFromLocalStorage, saveTasksToLocalStorage } from '../../../utils/LocalStorageUtil';
+
+
+export default function CompletedTask({ tasks, setTasks, handleDeleteTask, selectedDate }) {
+    
+    useEffect(() => {
+        const formattedDate = selectedDate.toISOString().split('T')[0]; // YYYY-MM-DD 형식
```

Comment:
요부분을 
```suggestion
        const formattedDate = format(selectedDate, 'yyyy-MM-dd');
```
로 바꾸는 건 어떠신가용?

Reply:
오 훨씬 깔끔한 것 같아요 감사합니다!